readme.txt

```diff
@@ -143,7 +143,7 @@ In the admin site, go to “Appearance / customize”, then select the “Additi
 .wmf-wp-with-preview::after {
     content: '';
 }
```

Comment:
Now that the new design is live I don't think we are making use of `::after` , I think that's a leftover from previous styling

Reply:
We should probably update this in [README.md](https://github.com/wikimedia/wikipediapreview-wordpress/blob/main/README.md) too

Reply:
checkout the latest css change here
https://github.com/wikimedia/wikipedia-preview/pull/174

Reply:
Oh got it! Nevermind then this looks good